ISISNeutronMuon/MDANSE
# Coordinate folding change

**Description of work**
- Change the default fold setting to false for all converters.
- Changed the folding so that the atom coordinates are moved to have a value between 0 and 1 in fractional coordinates.
- Added the fold setting to the ASE and CP2K jobs so that all trajectory converters have this option.

DFTB H2O trajectory converted with the fold setting on.
![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/6522eaeb-3fd4-4dc5-abbb-a0c69c565a95)


**Fixes**
Fixes #372

**To test**
Check that the fold option is available for the ASE and CP2K conversion jobs in the GUI. Convert a trajectory that does not have all atom position inside the unit cell (DFTB H2O for example) with the fold setting on and check that they get folded into the unit cell in the 3D view.
